.github/workflows/OCV-Coverage-4.x-U20.yaml

```diff
@@ -0,0 +1,295 @@
+      run: git clone --branch ${{ env.TARGET_BRANCH_NAME }} --reference ${{ env.GIT_CACHE_DOCKER }}/opencv.git https://github.com/opencv/opencv.git ${{ env.OPENCV_DOCKER_WORKDIR }}
+    - name: Clone opencv_extra
+      timeout-minutes: 60
+      run: git clone --single-branch --branch ${{ env.TARGET_BRANCH_NAME }} --reference ${{ env.GIT_CACHE_DOCKER }}/opencv_extra.git https://github.com/opencv/opencv_extra.git /home/ci/opencv_extra
+    - name: Extra DNN models update
+      timeout-minutes: 60
+      run: |
+        DOWNLOAD_MODELS_FILE='download_models.py'
+        LATEST_SAVED_HASH=$(cat ${{ env.DNN_MODELS }}/dnn/latest-hash-${{ env.TARGET_BRANCH_NAME }}.txt)
+        LATEST_HASH=$(sha256sum /home/ci/opencv_extra/testdata/dnn/$DOWNLOAD_MODELS_FILE | awk '{print $1}')
+        if [[ $LATEST_HASH == $LATEST_SAVED_HASH ]]; then
+          echo "DNN models are up to date"
+          echo "OPENCV_DNN_TEST_DATA_PATH=${{ env.DNN_MODELS }}" >> $GITHUB_ENV
+        else
+          echo "Updating DNN models list"
+          echo "OPENCV_DNN_TEST_DATA_PATH=/home/ci/new-dnn-models" >> $GITHUB_ENV && OPENCV_DNN_TEST_DATA_PATH='/home/ci/new-dnn-models'
+          mkdir -p "$OPENCV_DNN_TEST_DATA_PATH"
+          rsync -a --exclude=$DOWNLOAD_MODELS_FILE ${{ env.DNN_MODELS }}/* $OPENCV_DNN_TEST_DATA_PATH
+          cp /home/ci/opencv_extra/testdata/dnn/download_models.py $OPENCV_DNN_TEST_DATA_PATH/dnn
+          cd $OPENCV_DNN_TEST_DATA_PATH/dnn && python3 download_models.py
+        fi
+    - name: Configure OpenCV
+      timeout-minutes: 60
+      run: |
+        cd /home/ci/build
+        cmake -G Ninja ${{ env.EXTRA_CMAKE_OPTIONS }} ${{ env.OPENCV_DOCKER_WORKDIR }}
+    - name: Build OpenCV
+      timeout-minutes: 240
+      id: build-opencv
+      run: |
+        cd /home/ci/build
+        ninja
+    - name: Clean coverage
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && rm -rf coverage_html tmp*.gcno
+    - name: Zero coverage
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && lcov --directory /home/ci/build --zerocounters
+    - name: Init coverage
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && lcov --directory /home/ci/build --capture --initial -o /home/ci/build/opencv_test_base.info
+    - name: Accuracy:calib3d
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && xvfb-run -a bin/opencv_test_calib3d --test_threads=2 --test_tag_skip=size_hd --configuration=Debug
+    - name: Accuracy:core
+      timeout-minutes: 60
+      if: ${{ always() && steps.build-opencv.outcome == 'success' }}
+      run: cd /home/ci/build && xvfb-run -a bin/opencv_test_core --test_threads=2 --test_tag_skip=size_hd --configuration=Debug
```

Comment:
> `--configuration=Debug`

AFAIK, there is no such option for tests

Reply:
Took this from BuildBot: https://pullrequest.opencv.org/buildbot/builders/master_coverage-lin64-debug/builds/100000/steps/test_core/logs/stdio

Reply:
It is for `run.py` script. Here it is not used - test binaries are launched directly.

Reply:
Removed it from all tests.